docs/installation/post-install.rst

```diff
@@ -1,9 +1,9 @@
 Post-Installation Procedure
```

Comment:
Not related to your changes, but I would remove sparse file example as we don't want to use it in prod ... and replace it with real device example.

Reply:
Will do when tackling #2117